app/src/main/java/com/example/advanceduiapp/ui/multiViewRecycler/MultipleViewRecyclerFragment.kt

```diff
@@ -0,0 +1,36 @@
+package com.example.advanceduiapp.ui.multiViewRecycler
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import com.example.advanceduiapp.R
+import com.example.advanceduiapp.adapters.MultipleViewRecyclerAdapter
+import com.example.advanceduiapp.data.ToastButtonData
+import com.example.advanceduiapp.databinding.FragmentMultipleViewRecyclerBinding
+
+class MultipleViewRecyclerFragment : Fragment() {
+
+    private var _binding: FragmentMultipleViewRecyclerBinding? = null
+    private val binding get() = _binding!!
+    private val viewModel: MultipleViewRecyclerViewModel by viewModels()
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentMultipleViewRecyclerBinding.inflate(inflater, container, false)
+        val root = binding.root
+        val itemsData = viewModel.itemsData + listOf(
+            ToastButtonData(
+                this.requireContext(),
+                getString(R.string.butcher_greeting)
+            ), ToastButtonData(this.requireContext(), getString(R.string.butcher_greeting))
```

Comment:
make `ToastButtonData` that  receives ` @StringRes val title: Int` and create those instances in viewModel as well